src/index.html

```diff
@@ -11,6 +11,42 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form 
+      class="search-form" 
+      action="/"
+      method="get"
+      data-qa="big"
+    >
+      <label for="search-input">
+        <input 
+          class="input1" 
+          type="text" 
+          name="search" 
+          id="search-input"
+          data-qa="keypress"
+          placeholder="Try &quot;Los Angeles&quot;"
+        >
+
+      </label>
+    </form>
+
+    <!-- <br> -->
+
+    <form class="search-form2" 
+      action="/"
+      method="get"
+      data-qa="small"
+      >
+      <label for="search-input2">
+        <input 
+          class="input2" 
+          type="text" 
+          name="search2" 
+          id="search-input2"
+          placeholder="Try &quot;Los Angeles&quot;"
+          data-qa="small"
+        >
+      </label>
+    </form>
   </body>
-</html>
+</html>
```

Comment:
add empty line at the end of file
https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file